learing_journal/templates/edit-entry.jinja2

```diff
@@ -2,14 +2,22 @@
 {% block body %}
 
 <div class="jumbotron">
-  <!-- Entry ID: {{ single_entry.id }} -->
-  <h3>edit-entry.jinja2</h3>
-  <h2>Edit Journal Entry {{ single_entry.id }}</h2>
+  <h1>edit-entry.jinja2</h1>
+  <!-- TODO: Wrap the back to home link in a button -->
+  <!-- <h2><a href='/'>Back to Home</a></h2> -->
   <div class="form-group">
-    <label for="comment">Journal Body:</label>
-    <textarea class="form-control" rows="5" id="body">{{ single_entry.body  | safe }}</textarea>
+    <form action="{{ request.route_url('update',id=single_entry.id) }}" method="POST">
+      <label for="title">Title</label><br/>
+      <textarea class="form-control" rows="1" maxlength="100" id="title" name="title">{{ single_entry.title | safe }}</textarea>
```

Comment:
[Step 2] You probably don't need a whole `textarea` tag for this. It'd probably be better served with `<input type="text">`